README.md

```diff
@@ -164,15 +164,6 @@ try {
 }
 ```
 
-## `releaseConnection`
```

Comment:
Are you saying this should never be called?

Reply:
Not at the top level.  With this update mysql-easier is wrapping the pool and not the connection, so when you get a connection with `getConnection` you also need to release that distinct connection with `connection.release()`.